src/app/client/src/app/modules/groups/components/activity/activity-list/activity-list.component.html

```diff
@@ -1,39 +1,76 @@
-<!--Show loader-->
-<div class="sb-card-grid my-24" *ngIf="showLoader">
-  <div class="sb-card-grid--item" *ngFor="let i of numberOfSections">
-    <sb-course-card [isLoading]="true"></sb-course-card>
+<div *ngIf="!viewAll">
+<div *ngFor="let list of activityList">
+  <div class="header" *ngIf="list?.items?.length">
+      <div class="title">
+    {{list?.title}}
+      </div>
   </div>
-</div>
 
-<!--Show Activity List-->
-<div *ngIf="!showLoader && activityList?.length" class="sb-card-grid my-24">
-  <div class="sb-card-grid--item relative" *ngFor="let activity of activityList;">
-    <sb-course-card (cardClick)="openActivity($event, activity)" [course]="activity" [section]="null" [isMenu]="groupData?.isAdmin"
-      [cardImg]="activity?.appIcon || './assets/images/book.png'" (menuClick)="getMenuData($event, activity)">
-    </sb-course-card>
-    <div class="kabab-menu-dropdown-content" *ngIf="showMenu && selectedActivity?.identifier === activity?.identifier">
-      <div class="list sb-color-error" (click)="toggleModal(true)">
-        <span>{{resourceService?.frmelmnts?.lbl?.removeActivity}}</span>
+  <!--Show loader-->
+  <div class="sb-card-grid" *ngIf="showLoader && list?.items?.length">
+      <div class="sb-grid--item" *ngFor="let i of (list?.items || numberOfSections) ">
+          <sb-course-card [isLoading]="true"></sb-course-card>
       </div>
-    </div>
   </div>
+
+  <!--Show Activity List-->
+  <!-- <div  *ngIf="list?.items?.length && !showLoader">
+     <sb-cards-grid *ngIf="list?.items?.length" [title]="list?.title" [contentList]="list?.items" [maxCardCount]="3" [isLoading]="!list?.items.length" [viewMoreButtonText]="'View more'"
+      (cardClick)="openActivity($event)"  [isMenu]="groupData?.isAdmin"  (menuClick)="getMenuData($event)" [cardType]="getType(list?.type)" (viewMoreClick)="viewAllContents(list)"></sb-cards-grid>
+      <div class="kabab-menu-dropdown-content" *ngIf="showMenu">
+              <div class="list sb-color-error" (click)="toggleModal(true)">
+                  <span>{{resourceService?.frmelmnts?.lbl?.removeActivity}}</span>
+              </div>
+          </div>
+  </div> -->
+
+  <div  *ngIf="list?.items?.length && !showLoader" class="sb-card-grid">    
+       <div class="sb-card-grid--item relative" *ngFor="let activity of list?.items">
+          <sb-course-card *ngIf="list?.title === 'Courses'" (cardClick)="openActivity($event, activity)" [course]="activity" [section]="null" [isMenu]="groupData?.isAdmin" [cardImg]="activity?.appIcon
+      || './assets/images/book.png'" (menuClick)="getMenuData($event, activity)">
```

Comment:
Start using constant variable for this './assets/images/book.png' defining in appConfig